src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,27 +9,266 @@ logo: cloudcannon
+
+For more detailed instructions, see [CloudCannon's Getting Started Guide](https://cloudcannon.com/documentation/guides/getting-started-with-cloudcannon/).
+
+## CloudCannon collections and schemas
+
+If you use [Astro's content collections](/en/guides/content-collections/), then you will be familiar with CloudCannon's concepts of collections (used for organization/navigation in your Site Dashboard) and schemas (used to define the format of new content entries).
+
+Your CloudCannon Site Dashboard allows you to organize your Astro project's pages and content into collections: groups of related files with a similar format. This allows you to see similar types of content together for ease of editing and makes your content files easy to navigate, sort, and filter.
+
+### Create a CloudCannon schema for a collection
+
+To ensure that the data properties of your CloudCannon entries match the Zod validation `schema` defined in your content collection, you can create a [CloudCannon schema](https://cloudcannon.com/documentation/articles/what-is-a-schema/) (a blank template document for creating a new entry). Creating a template schema can ensure that any new documents created in CloudCannon will have the properties required by your content collection and avoid type errors in your project. Your CloudCannon schema can also include default values to start new documents, such as an author name for a single person blog.
+
+The following example will create a CloudCannon schema based on an Astro content collection (`blog`) for blog posts written in Markdown. This schema will be available when [creating a new entry](#creating-a-new-entry) from the CloudCannon "Posts" collection:
+
+<Steps>
+
+1. Create a folder at `.cloudcannon/schemas/` if it does not already exist.
+2. Add an existing blank file in this folder to be used as a default blog post template. The name is unimportant, but the file must have the same file extension as your Astro content collection entries (e.g. `post.md`).
+3. Provide the necessary frontmatter fields required by your content collection's schema. You do not need to provide any values for these, but any content you do include will be automatically included when a new entry is created. These are fields that will be available in the sidebar of your Content Editor.
+
+      The following example schema for a blog post has placeholders for the title, author, and date:
+
+      ```markdown title=".cloudcannon/schemas/post.md"
+      ---
+      title:
+      author:
+      date:
+      ---
+      
+      ```
+4. In your Cloudflare configuration file's `collections_config` property, add the file path to your schema under the CloudCannon collection under the "Posts" collection.
```

Comment:
wrong company 😅 
```suggestion
4. In your CloudCannon configuration file's `collections_config` property, add the file path to your schema under the CloudCannon collection under the "Posts" collection.
```

Reply:
If I only did this once, I'll take it! (I had to correct myself SO MANY TIMES!!)